### ISSUE TITLE
Cant found main class gods.game.GodsGame in linux

### ISSUE THREAD

**Issue by orderorder** [jotd666/gods-deluxe#2]
I have installed "jre-openjdk 15.0.2.u7-1"

After ./gods.sh

Cant found or load main class gods.game.GodsGame 

cause: java.lang.ClassNotFoundException: gods.game.GodsGame

**Comment by jotd666** [comment]
Hi,

Yeah, github archive only contains the source code.

If you're not interested in building it, you can grab a pre-built archive at https://jotd.pagesperso-orange.fr/gods/

**Comment by orderorder** [comment]
Thanks for this project. Will you continue to add improvements in the future?

**Comment by jotd666** [comment]
well, I don't think so now. This is pretty much complete and I'm quite bored with that (after 14 years of hacking through it). But pull requests are certainly welcome to improve it :)

**Comment by mbugni** [comment]
@jotd666 are you still bored? [I hope no](https://github.com/jotd666/gods-deluxe/issues/4).

@orderorder maybe one day there will be a simpler way to [play this on Linux](https://github.com/jotd666/gods-deluxe/issues/4).

**Comment by mbugni** [comment]
Please give a try to [Release 0.9.0](https://github.com/jotd666/gods-deluxe/releases/tag/0.9.0). It requires Java 8 as minimum.

**Comment by mbugni** [comment]
> Do you plan to move it to another programming language?

Not from my side.

> What plans do you have for this project in the future?

In a short term, I'm planning to publish it on [Flathub](https://flathub.org/).

@orderorder does it work for you? You should just run the gods.sh command...

**Comment by orderorder** [comment]
Yes, it works for me but at some moments the character gets stuck on the stairs. When I have time I will test it more thoroughly and if necessary I will report errors or opinions to you.

**Comment by jotd666** [comment]
Please provide the steps to reproduce. I never got that bug. I got tons of bugs but not that one

**Comment by mbugni** [comment]
> The editor was used to create custom levels for the original Magic Pockets game though!

@jotd666 did you ever consider to adopt something standard [like this](https://www.mapeditor.org/)? Generally speaking, moving the game to standard frameworks/libraries...

**Comment by mbugni** [comment]
> Yes, it works for me but at some moments the character gets stuck on the stairs

Please, also give a try to the [Flatpak version](https://flathub.org/it/apps/io.github.jotd666.gods-deluxe).
